deps/jerry/jerryscript.gyp

```diff
@@ -52,6 +52,8 @@
         'JERRY_GLOBAL_HEAP_SIZE=16384',
         'JERRY_LINE_INFO=1',
         'JERRY_DEBUGGER=0',
+        'JERRY_MEM_STATS=0',
+        'JERRY_LOGGING=1',
```

Comment:
Can it copy the value of JERRY_MEM_STATS?

Reply:
Good Idea!